### ISSUE TITLE
Incorrect ignore-case search

### ISSUE THREAD

**Issue by coderazzi** [coderazzi/tablefilter-swing#87]
(Original issue 1 created by {u'display_name': u'coderazzi coderazzi', u'account_id': u'557058:790eb198-2ae4-49b4-b920-8d654aa86b3f'} on 2010-05-21T01:26:48+00:00)

```
If a column contains some entries, like Bua... and BUb..., and ignore case
is set to true
a) the ordering is wrong: BUb... will come before Bua...
b) Looking for Bua renders no results

```

Original issue reported on code.google.com by `coderazzi` on 2010-05-21 01:26:48

**Comment by coderazzi** [comment]
(Original issue 1 last updated on 2015-04-26T00:27:18.995420+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by coderazzi** [comment]
(Original comment by {u'display_name': u'coderazzi coderazzi', u'account_id': u'557058:790eb198-2ae4-49b4-b920-8d654aa86b3f'} on 2010-05-24T13:47:32+00:00)

```
Solved on release 3.1.2
```

Original issue reported on code.google.com by `coderazzi` on 2010-05-24 13:47:32
